src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs

```diff
@@ -162,6 +199,19 @@ public OperationResultResponse<ProjectInfo> Execute(ProjectRequest request)
 
             _validator.ValidateAndThrowCustom(request);
 
+            var existUsers = CheckUserExistence(request.Users.Select(u => u.UserId).ToList(), response.Errors);
+            if (!response.Errors.Any()
+                && existUsers.Count() != request.Users.Count())
+            {
+                response.Status = OperationResultStatusType.PartialSuccess;
```

Comment:
Not need in this line, in the end of command you set the status(line 241)

Reply:
done